rpcs3/Emu/Cell/PPUTranslator.cpp

```diff
@@ -4712,7 +4712,7 @@ void PPUTranslator::SetFPCC(Value* lt, Value* gt, Value* eq, Value* un, bool set
 
 void PPUTranslator::SetFPRF(Value* value, bool set_cr)
 {
-	//const bool is32 =
+	const bool is32 =
```

Comment:
for readability I would just write:
```suggestion
	ensure(value->getType()->isFloatTy() || value->getType()->isDoubleTy());
```

Reply:
Well, could use maybe_unused but seems working like this, just like the rest of the function is commented out.